src/report/report-system/html-barchart.scm

```diff
@@ -37,6 +37,8 @@
 		      legend-reversed?
                       row-labels-rotated?
 		      stacked?
+                      pixels-width?
+                      pixels-height?
```

Comment:
Building further on the idea of using a pair to store the width and height information (as in "(absolute . 400)" or "(relative . 100)"), there is no need to add these two extra parameters. Instead, the information you are looking for is directly encoded in the width and height parameters passed to this module.